docs/_includes/Running-Spark-on-Alluxio/fault-tolerant-mode-with-zookeeper-bash.md

```diff
@@ -1,7 +1,4 @@
 ```bash
-export SPARK_JAVA_OPTS="
-  -Dalluxio.zookeeper.address=zookeeperHost1:2181,zookeeperHost2:2181
-  -Dalluxio.zookeeper.enabled=true
-  $SPARK_JAVA_OPTS
-"
+spark.driver.extraJavaOptions -Dalluxio.zookeeper.address=zookeeperHost1:2181,zookeeperHost2:2181 -Dalluxio.zookeeper.enabled=true
```

Comment:
I have the same question. Do we need quotes around all the options, or does this still work?